core/time/shared/gmtime.rb

```diff
@@ -3,8 +3,10 @@
     # Testing with America/Regina here because it doesn't have DST.
     with_timezone("CST", -6) do
       t = Time.local(2007, 1, 9, 6, 0, 0)
+      t.should_not.utc?
```

Comment:
Not sure about this check. It actually checks that the `with_timezone` works fine, as far as it should switch the global timezone of Ruby VM to the specified timezone/offset.

https://github.com/ruby/mspec/blob/5354a405e327bb144f00157b913aa544c3b77e4b/lib/mspec/helpers/datetime.rb#L27-L48

Reply:
This is not meant as a check by itself, but I often add these kind of things to illustrate the before/after difference. Befaore the call to `Time#gmtime`, the variable is not UTC, but after the call it is.
So it's more of a documentation thing than a validation thing.

Reply:
Anyway, after the rewrite following the previous suggestion, it's gone since it was no longer relevant.